.circleci/config.yml

```diff
@@ -211,10 +210,9 @@ jobs:
     steps:
       - checkout
       - attach_workspace:
-          at: /go/bin
+          at: /home/circleci/go/bin
       - run: sudo apt-get update --allow-releaseinfo-change-suite --allow-releaseinfo-change-version && sudo apt-get install -y rsyslog
       - run: sudo service rsyslog start
-      - run: *install-gotestsum
```

Comment:
Are we removing the gotestsum install step here ?

Reply:
@kisunji mentioned this in the other PR, but this is already installed as part of `cimg/go`